Modules/Module_Processing/Module_ASL/xASL_wrp_RegisterASL.m

```diff
@@ -217,10 +213,16 @@
 
 
 %% E. Allow registration without structural data
+%% PM: This part is partly equal to part xASL_module_ASL > section A2
 StructuralDerivativesExist = xASL_exist(x.P.Path_y_T1, 'file') && xASL_exist(x.P.Path_c1T1, 'file') && xASL_exist(x.P.Path_c2T1, 'file');
 StructuralRawExist = xASL_exist(x.P.Path_T1, 'file') || xASL_exist(x.P.Path_T1_ORI, 'file');
 
-% In case that we don't have the structural derived data, we need to check the reason
+% By default, don't use dummy structural even if the structural image is missing

```

Comment:
Since we are modifying this, wouldn't it make more sense to move this to the ASL module where we deal with all dataPar parameters?!

Reply:
Not sure, we have 6 parts A-6 that are more "special/specific" that parsing ASL processing parameters.
This is about if we will run xASL_module_ASL yes or no (in section A2 in xASL_module_ASL).

Here in xASL_wrp_RegisterASL, we repeat this for the case that somebody runs this wrapper separately or does debugging. I thought about standardizing it, but it the sections in both functions are not completely identical; they have a different function.

Reply:
Sure - for debugging and separate execution, we can enable/disable the boolean. 
But all dataPar processing booleans are initialized in ASL module. It makes no sense to do it elsewhere. I of course only talk about this boolean.

Reply:
I didn't change that. I only improved its behavior here and at the start of the ASL module.

Reply:
Fixed in 0248a3ea5f9cce491b593c7befbbaf5af5055b64